### PR TITLE
docs: Add missing steps to running locally

### PR DESCRIPTION
**Pull request by gabalafou**
The repo readme was missing instructions for running locally:

1. Cloning from GitHub
2. Installing via npm
3. Configuring environment (.env files)

### CODE REVIEW

**Review by galdranorn — APPROVED** [review]

**Comment by gabalafou** [comment]
@galdranorn thanks for approving! Can I go ahead and squash-merge this into the develop branch? Every check is passing except the Vercel Quansight Labs deployment, but that can't be related to this PR.

File: README.md [hunk 1]
```diff
@@ -2,7 +2,42 @@
 
 ## Running project locally
 
-Run `npm run start:consulting` or `npm run start:labs` to start a corresponding dev server. Navigate to http://localhost:4200/ or use localhost preview in Storyblok panel. On the localhost the app will automatically reload if you change any of the source files, in the Storyblok panel you need to refresh the page manually.
+Prerequisites: Node and npm.
```
Comment (trallard) [inline README.md:5]: Please add links for lazy folks like me

File: README.md [hunk 2]
```diff
@@ -2,7 +2,42 @@
...
+- [consulting environment
+  variables](https://vercel.com/quansight/quansight-consulting/settings/environment-variables)
+- [labs environment
+  variables](https://vercel.com/quansight/quansight-labs/settings/environment-variables)
+
+If the example .env file above does not match what you read in the Vercel admin,
+use what's in Vercel.
+
+Run `npm run start:consulting` or `npm run start:labs` to start a corresponding
+dev server. Navigate to http://localhost:4200/ or use localhost preview in
```
Comment (trallard) [inline README.md:32]:
```suggestion
dev server. Navigate to <http://localhost:4200/> or use localhost preview in
```

**Review by trallard — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (marcin-antczak-naturaily) [reply README.md:5]: Done. No need to mention NPM as it's a part of NodeJS installation package.

**Pull request merged**